server/bin/unittests

```diff
@@ -477,6 +478,7 @@ function _reset_state {
     export PATH=${_orig_PATH}
     unset CONFIG
     unset LOGSDIR
+    unset STATECONFIG
```

Comment:
This does nto remove the file: it just unsets the env variable you defined on l. 307 (which btw is not used anywhere, so there no reason to define it).

Reply:
It's the file that's (possibly) created on l.325 that need to be removed in the section below: you need to do

      rm -f ${_testopt}/lib/config/state-pbench-server.cfg

This will work whether the file exists or not: the -f suppresses the error message if it does not exist.